librz/core/cfile.c

```diff
@@ -563,16 +533,14 @@ CURSOR cursor position(offset from curseek)
 	sdb_sync(config_sdb);
 	sdb_free(config_sdb);
 	rz_sys_setenv("RZ_CONFIG", config_sdb_path);
-
 	rz_sys_setenv("RZ_BIN_LANG", rz_config_get(core->config, "bin.lang"));
 	rz_sys_setenv("RZ_BIN_DEMANGLE", rz_config_get(core->config, "bin.demangle"));
 	rz_sys_setenv("RZ_ARCH", rz_config_get(core->config, "asm.arch"));
-	rz_sys_setenv("RZ_BITS", sdb_fmt("%" PFMT64u, rz_config_get_i(core->config, "asm.bits")));
-	rz_sys_setenv("RZ_COLOR", rz_config_get_i(core->config, "scr.color") ? "1" : "0");
-	rz_sys_setenv("RZ_DEBUG", rz_config_get_b(core->config, "cfg.debug") ? "1" : "0");
-	rz_sys_setenv("RZ_IOVA", rz_config_get_i(core->config, "io.va") ? "1" : "0");
+	rz_sys_setenv("RZ_BITS", rz_config_get(core->config, "asm.bits"));
+	rz_sys_setenv("RZ_COLOR", rz_config_get(core->config, "scr.color"));
+	rz_sys_setenv("RZ_DEBUG", rz_config_get(core->config, "cfg.debug"));
+	rz_sys_setenv("RZ_IOVA", rz_config_get(core->config, "io.va"));
```

Comment:
This is not exactly the same as before.. For example, for booleans vars it would returne `true`/`false` instead of `1`/`0` as before. I think for env vars it is more common to have 1/0 ?

Reply:
no. this better and this stuff is not used besides the user itself.

Reply:
How is this better exactly? Anyway, I'm ok with having `true`/`false` instead of 1/0, but please update the documentation because it says 1/0.

Reply:
book needs to be updated anyway after this PR

Reply:
I meant the rizin doc for `%??`.

By the way, I changed my mind. I think there is no need to do this change here and now. In particular, there are places that seem to assume boolean env vars as 1/0 and not true/false,  so I'll revert these.